benchmark/src/main/java/ai/timefold/solver/benchmark/impl/SolverBenchmarkFactory.java

```diff
@@ -91,6 +91,8 @@ protected List<SolverMetric> getSolverMetrics(ProblemBenchmarksConfig config) {
                 .orElseGet(ProblemStatisticType::defaultList)) {
             if (problemStatisticType == ProblemStatisticType.SCORE_CALCULATION_SPEED) {
                 out.add(SolverMetric.SCORE_CALCULATION_COUNT);
+            } else if (problemStatisticType == ProblemStatisticType.MOVE_CALCULATION_SPEED) {
```

Comment:
Question: does "move calculation" make sense? Move's aren't calculated.
Arguably "move execution" would be more correct here.
Any other ideas?

Reply:
What about `MOVE_COUNT_SPEED`?

Reply:
![image](https://github.com/user-attachments/assets/4360d4b1-f558-46d7-aae2-1aea40775442)
![image](https://github.com/user-attachments/assets/b923a373-9452-45a0-aa6a-e40d0fabb702)
![image](https://github.com/user-attachments/assets/c087aeca-22e0-4799-8c3c-be1399f8f97b)